### ISSUE TITLE
Can a relative pointer be resolved to actual value?

### ISSUE THREAD

**Issue by netcafa** [josdejong/jsoneditor#310]
Hello all,

For example,
If Enum definition contains a relative pointer to one array, this Enum will not be resolved to actual array.
That means the enum list can not be showed in jsoneditor tree view.

**Comment by josdejong** [comment]
Is this the same as #302?

**Comment by netcafa** [comment]
Yes. what i mentioned is like the following, but I think they are the same.  Please ignore it.
```
    var schema = {
      "properties": {
        "allowedValues": {},
        "value": { "enum": { "$data": "1/allowedValues" } }
      }
    };
```

**Comment by netcafa** [comment]
Yes. what i mentioned is like the following, but I think they are the same. Please ignore it.

    var schema = {
      "properties": {
        "allowedValues": {},
        "value": { "enum": { "$data": "1/allowedValues" } }
      }
    };

**Comment by josdejong** [comment]
Ah, I get it, thanks. Would indeed be nice if JSONEditor would handle this too.
